.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing
+
+There's no such thing as a small contribution, anyway you can help improve our website is welcome. All we ask is that we are a volunteer of [Women Hack For Non-Profits](http://www.womenhackfornonprofits.com/volunteer-contact.html) and you identify yourself as a woman.
+
+### Who are [Women Hack For Non-Profits](http://www.womenhackfornonprofits.com/)
+Women Hack for Non-Profits matchmakes skilled women in technology to open source projects for non-profits, charities and individuals with a good social cause. By doing so, we aim to enhance the skill set of Women in Tech to support their growth and increase their footprint in the open source community.
+
+### How can I contribute?
```

Comment:
Nope. It needs to be decided amongst the team and we can add them after.

I'll create an issue for that
On Thu, 12 May 2016 at 23:26, Liliana Kastilio <notifications@github.com>
wrote:

> In .github/CONTRIBUTING.md
> <https://github.com/womenhackfornonprofits/whfnp/pull/91#discussion_r63107901>
> :
>
> > @@ -0,0 +1,39 @@
> > +# Contributing
> > +
> > +There's no such thing as a small contribution, anyway you can help improve our website is welcome. All we ask is that we are a volunteer of [Women Hack For Non-Profits](http://www.womenhackfornonprofits.com/volunteer-contact.html) and you identify yourself as a woman.
> > +
> > +### Who are [Women Hack For Non-Profits](http://www.womenhackfornonprofits.com/)
> > +Women Hack for Non-Profits matchmakes skilled women in technology to open source projects for non-profits, charities and individuals with a good social cause. By doing so, we aim to enhance the skill set of Women in Tech to support their growth and increase their footprint in the open source community.
> > +
> > +### How can I contribute?
>
> Did we have any guidelines in terms if code style/what sort of changes are
> okay/not okay to do?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/womenhackfornonprofits/whfnp/pull/91/files/b8c04c43470e3d79827ca4e90491be09ea78cdc5#r63107901>
>
-- 

Tanya Powell
about.me/tanyapowell
<https://about.me/tanyapowell?promo=email_sig>